package.json

```diff
@@ -73,7 +73,7 @@
     "eslint-plugin-jsx-a11y": "6.1.2",
     "eslint-plugin-prettier": "3.0.0",
     "eslint-plugin-react": "7.11.1",
-    "flow-bin": "0.85.0",
+    "flow-bin": "^0.85.0",
```

Comment:
hmm wondering how this one sneaked in.

Reply:
hmm.. I did `npm install flow-bin` .. will fix it in a second

Reply:
updated here 773ccda